File: programme/index-of-alphabet/index-of-alphabet.cpp
// Write a programme to find index of an alphabet
#include <iostream>

using namespace std;

int findIndex(char input);

int findIndex(char input)
{
    int ascii_value = (int)input;
    if (ascii_value > 95)
        return (ascii_value - 96);
    else
        return (ascii_value - 64);
}

int main()
{
    cout << "Input  : ";
    char input;
    cin >> input;

    cout << "Output : " << findIndex(input) << endl;
}

File: programme/index-of-alphabet/index-of-alphabet.jl
# Write a programme to find index of an alphabet

function chars(char)
  char = Int(char)
  
  if char >= 97
    small = Int('a')
    println(char - small + 1)
  else
    capital = Int('A')
    println(char - capital + 1)
  end
  
end

input = 'C'
println("Input  : ", input)
print("Output : ")
chars('C')
